media/css/newsletter/newsletter-management.scss

```diff
@@ -29,7 +26,7 @@ main {
     }
 
     .mzp-c-field {
-        padding-bottom: $label-v-spacing;
+        padding-bottom: form.$label-v-spacing;
```

Comment:
I kind of like keeping the namespace to indicate these variables are separate from the `lib` import, but we could wildcard alias these as well if preferred